### DISCUSSION TITLE
New to GitHub

### DISCUSSION THREAD

**Discussion by HammadJahangir123** [community/community#110312] [New to GitHub]
Hello and I am new GitHub community and Please Help me. How can use this community?

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @HammadJahangir123  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
